.github/workflows/scans.yml

```
@@ -168,12 +168,14 @@ jobs:
       - name: Stats page
         continue-on-error: true
         timeout-minutes: 10
-        uses: "MTES-MCT/stats-action@main"
+        uses: "betagouv/check-url-action@main"
         if: ${{ matrix.sites.tools.stats }}
         with:
           url: ${{ matrix.sites.url }}
-          uri: 'stats'
-          output: scans/stats.json
+          uri: stats
+          output: stats.json
```

```suggestion
          output: scans/stats.json
```

il faut que ce soit dans le dossiers scans pour être automatiquement comité à la fin